ecosystem/sep-0024.md

```diff
@@ -768,6 +768,7 @@ wallet. For example:
 ## Info
 
 Allows an anchor to communicate basic info about what their `TRANSFER_SERVER_SEP0024` supports to wallets and clients.
+If an anchor supports XLM, the `native` asset code should be included in the response.
```

Comment:
I understand it is useful to include XLM here to make sure folks know what we're talking about when we say native. It might be worth tweaking this text so that the focus is on the native asset, and then we have a note educating what the native asset is called on pubnet. Suggest something like:
```suggestion
If an anchor supports the `native` asset, the `native` asset code should be included in the response.

The `native` asset on the Stellar Public Network is the [lumen (XLM)](lumen).

[lumen]: https://stellar.org/lumens
```